app/styles/modules/_settings-totp.scss

```diff
@@ -138,6 +138,15 @@
   margin-top: 25px;
 }
 
+.save-option-button {
```

Comment:
If we add these styles to the [avatar change modal as specified in the original issue](https://github.com/mozilla/fxa-content-server/issues/6257), these styles will need to be extracted into a shared location. I suggest somewhere in [_button-row.scss](https://github.com/AlbionaHoti/fxa-content-server/blob/54cc73cdda13648e94061671dcaec97c352f1d16/app/styles/modules/_button-row.scss#L3).